edgelet/iotedge/src/config/apply.rs

```diff
@@ -32,7 +32,7 @@ pub fn execute(config: &Path) -> Result<(), std::borrow::Cow<'static, str>> {
             get_system_user("aziotks").map_err(|err| format!("{}", err))?,
             get_system_user("aziotcs").map_err(|err| format!("{}", err))?,
             get_system_user("aziotid").map_err(|err| format!("{}", err))?,
-            get_system_user("aziotpm").map_err(|err| format!("{}", err))?,
+            get_system_user("aziottpm").map_err(|err| format!("{}", err))?,
```

Comment:
If TPM is not used, it will throw error. is that what we want?

Reply:
Currently IIS Creates a user regardless of whether TPM is present or not